docs/src/pages/[platform]/getting-started/usage/index.page.mdx

```diff
@@ -0,0 +1,10 @@
+---
+title: Usage
+metaTitle: Usage
```

Comment:
nit: if the `metaTitle` is the same as `title`, we don't have to add `metaTitle` .

Same for docs/src/pages/[platform]/getting-started/installation/index.page.mdx

Reply:
Thanks, updated!